ports/imgui/CMakeLists.txt

```
@@ -6,135 +6,225 @@ set(CMAKE_DEBUG_POSTFIX d)
 add_library(${PROJECT_NAME} "")
 add_library(${PROJECT_NAME}::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
 target_include_directories(
-	${PROJECT_NAME}
-	PUBLIC
-		$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
-		$<INSTALL_INTERFACE:include>
+    ${PROJECT_NAME}
+    PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+        $<INSTALL_INTERFACE:include>
 )
 
 target_sources(
-	${PROJECT_NAME}
-	PRIVATE
-		${CMAKE_CURRENT_SOURCE_DIR}/imgui.cpp
-		${CMAKE_CURRENT_SOURCE_DIR}/imgui_demo.cpp
-		${CMAKE_CURRENT_SOURCE_DIR}/imgui_draw.cpp
-		${CMAKE_CURRENT_SOURCE_DIR}/imgui_widgets.cpp
-		${CMAKE_CURRENT_SOURCE_DIR}/misc/cpp/imgui_stdlib.cpp
+    ${PROJECT_NAME}
+    PRIVATE
+        ${CMAKE_CURRENT_SOURCE_DIR}/imgui.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/imgui_demo.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/imgui_draw.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/imgui_widgets.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/misc/cpp/imgui_stdlib.cpp
 )
 
 
 if(IMGUI_USE_WCHAR32)
-	FILE(READ ${CMAKE_CURRENT_SOURCE_DIR}/imconfig.h IMCONFIG)
-	STRING(REGEX REPLACE "//#define IMGUI_USE_WCHAR32" "#define IMGUI_USE_WCHAR32" IMCONFIG "${IMCONFIG}")
-	FILE(WRITE ${CMAKE_CURRENT_SOURCE_DIR}/imconfig.h "${IMCONFIG}")
+    FILE(READ ${CMAKE_CURRENT_SOURCE_DIR}/imconfig.h IMCONFIG)
+    STRING(REGEX REPLACE "//#define IMGUI_USE_WCHAR32" "#define IMGUI_USE_WCHAR32" IMCONFIG "${IMCONFIG}")
+    FILE(WRITE ${CMAKE_CURRENT_SOURCE_DIR}/imconfig.h "${IMCONFIG}")
 endif()
 
-if(IMGUI_BUILD_ALLEGRO5_BINDING)
-	find_path(ALLEGRO5_INCLUDE_DIRS allegro5/allegro.h)
-	target_include_directories(${PROJECT_NAME} PRIVATE ${ALLEGRO5_INCLUDE_DIRS})
-	target_sources(${PROJECT_NAME} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/examples/imgui_impl_allegro5.cpp)
+set(BINDING_TARGETS )
+set(BINDINGS_SOURCES)
+if(IMGUI_INSTALL_ALLEGRO5_BINDING)
+    find_path(ALLEGRO5_INCLUDE_DIRS allegro5/allegro.h)
+    add_library(imgui_impl_allegro5 INTERFACE)
+    set_property(TARGET imgui_impl_allegro5 PROPERTY INTERFACE_SOURCES $<INSTALL_INTERFACE: examples/imgui_impl_allegro5.cpp>)
```

```suggestion
    set_property(TARGET imgui_impl_allegro5 PROPERTY INTERFACE_SOURCES $<INSTALL_INTERFACE:examples/imgui_impl_allegro5.cpp>)
```

That extra space shouldn't be in any of these.